Makefile

```diff
@@ -75,6 +84,9 @@ clean:
 	rm -rf build
 	rm -rf node_modules
 
+#clean.tycho: remove tycho
+clean.tycho:
```

Comment:
Can you add this to `clean` so it's called automatically?

Reply:
Yes, i initially thought of adding this to clean. But then again while reinstall, the changes to tycho would be lost and assumed it would be nice to separate these out.

I agree with adding this to clean as well. Will go ahead and make that change.